# this message type is to store miscellaneous packbot parametes
# in this case,
# 	1) status of the attack light
#		2) drive camera tilt degrees
# attack light
bool light
# drive camera tilt angle in degrees
# unit - degrees, integer value
int32 camera_tilt
